modules/helm-tls/main.tf

```diff
@@ -0,0 +1,150 @@
+# ------------------------------------------------------------------------------
+# TERRAFORM / PROVIDERS CONFIG
+# ------------------------------------------------------------------------------
+
+terraform {
+  backend "gcs" {}
+}
+
+provider "tls" {}
+
+# ------------------------------------------------------------------------------
+# CA / ROOT CERTIFICATE
+# ------------------------------------------------------------------------------
+
+locals {
+  tls_dir       = "${var.custom_tls_dir == "" ? var.tiller_namespace : var.custom_tls_dir}"
```

Comment:
This is not consistent with the other module: `tls_dir       = "${var.secrets_dir}/${var.custom_tls_dir == "" ? var.tiller_namespace : var.custom_tls_dir}/${var.helm_dir_name}"`

Reply:
Nice catch! Updated.